docs/src/man/getting_started.md

```diff
@@ -128,6 +128,14 @@ dt = DataTable(A = 1:4, B = randn(4))
 colwise(c->cumsum(dropnull(c)), dt)
 ```
 
+
+## IO
+
+For reading and writing tabular data, load the [CSV.jl](https://github.com/JuliaData/CSV.jl) package with
+```julia
+using CSV
+```
+
 ## Accessing Classic Data Sets
```

Comment:
Actually, I think the R datasets don't apply to this package, since RDatasets.jl only creates `DataFrame`s AFAIK.